publiccode.yaml

```diff
@@ -0,0 +1,50 @@
+publiccodeYmlVersion: "0.2"
+name: Signalen frontend
+url: "https://github.com/signalen/frontend"
+softwareVersion: "1"
+releaseDate: "2021-05-06"
+platforms:
+  - web
+categories:
+  - it-development
+usedBy:
+  - Gemeente Amsterdam
+developmentStatus: development
+softwareType: standalone/web
+description:
+  nl:
+    localisedName: Signalen frontend
+    genericName: Signalen frontend
+    shortDescription: >-
+      This project provides a web frontend for Signalen, an application that helps cities manage and prioritize nuisance reports.
+    longDescription: >-
+      This project provides a web frontend for Signalen, an application that helps cities manage and prioritize nuisance reports.
+    documentation: "https://github.com/Signalen/frontend/blob/develop/README.md"
```

Comment:
Is https://docs.signalen.org/ a better documentation link?

Reply:
I think so